src/components/Books.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Books = () => {
+	const books = [
```

Comment:
* I suggest you create a book component to be imported into Books.jsx for rendering each book which would also have a dummy button (remove) which isn't expected to work for now [Use the project big picture as guide ](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek.md)